sphinxext/opengraph/event_handler.py

```diff
@@ -0,0 +1,15 @@
+from typing import Dict, Any
+from sphinx.application import Sphinx
+from .docutils import nodes
+from .util import insert_tags
+
+
+def html_page_context(
+    app: Sphinx,
+    pagename: str,
+    templatename: str,
+    context: Dict[str, Any],
+    doctree: nodes.document,
+) -> None:
+    if doctree:
+        insert_tags(context, doctree, app.config)
```

Comment:
`html_page_context` can be combined with `insert_tags` since it (`html_page_context`) is just one if statement.

Reply:
Probably, this was that way is was originally so I just kinda left it that way, will change